docs/docs-graphql/schema.graphqls

```diff
@@ -3968,6 +3968,11 @@ type Service {
     kill: Boolean = false
   ): ServiceID!
 
+  """Forces evaluation of the pipeline in the engine."""
+  sync: ServiceID!
```

Comment:
Why was this added anyway? Seems like it's a no-op? (`Evaluate` just returns `nil, nil`)

Reply:
This just "forces" it to be sent to the engine.

```go
// this never gets sent to the engine, since there's never a query
LoadServiceFromID(id).Terminal()

// i need some method to actually "force" it to be sent
LoadServiceFromID(id).Terminal().Sync(ctx)
```

*Maybe* `Service.terminal` could always take a `ctx`, similar to `Start`? But then it has a different signature than `Container.terminal`/`Directory.terminal`.